website/src/content/docs/docs/emitters/sdk-clients/introduction.md

```diff
@@ -0,0 +1,121 @@
+---
+title: introduction of SDK client emitters
+---
+
+## How to Use Client Emitters to Generate SDK from TypeSpec
+
+### Introduction
+
+This guide will walk you through the process of using different client emitters (JavaScript, Python, Java, .NET) to generate SDKs from TypeSpec. Please note that all client emitters are currently in **preview** and are subject to changes in future versions.
+
+By following this guide, you will learn:
+
+1. How to set up and configure client emitters.
+2. Update the client emitter configurations in `package.json` and `tspconfig.yaml`.
+3. How to run the SDK generation for each specific programming language.
+
+## Location of All Client Emitters and Common Configurations
+
+The client emitters and their common configurations are located in the `package.json` file within your project.
+
+| **Emitter Name**             | **Language** | **Version**              | **Common Configuration** |
+| ---------------------------- | ------------ | ------------------------ | ------------------------ |
+| @azure-tools/typespec-ts     | JavaScript   | `0.38.1`                 | `emitter-output-dir`     |
+| @typespec/http-client-python | Python       | `0.6.6`                  | `emitter-output-dir`     |
+| @typespec/http-client-java   | Java         | `0.1.9`                  | `emitter-output-dir`     |
+| @typespec/http-client-csharp | .NET         | `0.1.9-alpha.20250113.2` | `emitter-output-dir`     |
+
+### Common Configuration Options
+
+- `emitter-output-dir`: Defines where the generated SDK files will be stored.
+
+Below is an example of the `package.json` snippet where client emitters are defined:
+
+```json
+  "dependencies": {
+    "@typespec/http-client-csharp": "^0.1.9-alpha.20250113.2",
+    "@typespec/http-server-javascript": "^0.58.0-alpha.6",
+    "@typespec/http-client-java": "^0.1.9",
+    "@typespec/http-client-python": "^0.6.6",
+    "@azure-tools/typespec-ts": "^0.37.0",
+  }
+```
+
+#### Note: Check for the Latest Version
+
+To ensure you are using the latest version of the packages, visit npmjs.com and search for the relevant packages.
+
+## Language-Specific Settings
+
+### JavaScript Client Emitter
+
```

Comment:
@weidongxu-microsoft, could you also include the required dependencies like JDK and Maven installation in this doc?

Reply:
Thanks for the remind. Added a section about required dependencies.